src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -100,12 +84,41 @@ private ArgumentCaptor(Class<? extends T> clazz) {
      * @return null or default values
      */
     public T capture() {
-        Mockito.argThat(capturingMatcher);
+        return captureIf(Any.ANY);
```

Comment:
Hi, 
For this use case we can use `captureIf`. 
Right now for `capture` we register an `ANY` matcher. We could register a `Null` matcher too. I don't see the need of `IsNullOrA`. What do you think?